.gitpod.yml

```diff
@@ -1,13 +1,15 @@
+---
 # List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
 tasks:
   - name: Install npm depdencies
     init: npm i
     command: |
-      gp await-port 27017
+      mv .env.example .env
```

Comment:
I'm renaming the example env here to a regular one, so the `dev` command executes as expected.

When not doing this, it probably doesn't make sense to run `dev` because the database won't connect.
I personally would go with the solution that is currently in that PR, what do you think?

Reply:
Sounds good 👍